MyLibrary/Sources/ScheduleFeature/Schedule.swift

```diff
@@ -38,6 +38,7 @@ public struct Schedule {
     case path(StackAction<Path.State, Path.Action>)
     case destination(PresentationAction<Destination.Action>)
     case view(View)
+    case fetchResponse(Result<(Conference, Conference, Conference), Error>)
```

Comment:
Might be better with creating struct.
```suggestion
    case fetchResponse(Result<SchedulesResponse, Error>)
```

The struct may be like below:

```swift
public struct SchedulesResponse: Equatable {
  var day1: Conference
  var day2: Conference
  var workshop: Conference
}